media-plugins/vdr-pvr350/vdr-pvr350-1.7.5.ebuild

```diff
@@ -14,27 +14,15 @@ SRC_URI="mirror://vdr-developerorg/${VERSION}/${P}.tgz"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="yaepg"
 
 DEPEND=">=media-video/vdr-2
 	media-sound/mpg123
 	media-sound/twolame
-	media-libs/a52dec
-	yaepg? ( >=media-video/vdr-2[yaepg] )"
+	media-libs/a52dec"
 RDEPEND="${DEPEND}"
 
-DEPEND="${DEPEND}
-	|| ( >=sys-kernel/linux-headers-2.6.38 )"
-
 S="${WORKDIR}/${P#vdr-}"
-
-pkg_setup() {
-	vdr-plugin-2_pkg_setup
-
-	if use yaepg; then
-		BUILD_PARAMS="SET_VIDEO_WINDOW=1"
-	fi
-}
+QA_FLAGS_IGNORED="usr/lib/vdr/plugins/libvdr-.* usr/lib64/vdr/plugins/libvdr-.*"
```

Comment:
put this on two lines for readability
```
QA_FLAGS_IGNORED="
	usr/lib/vdr/plugins/libvdr-.*
	usr/lib64/vdr/plugins/libvdr-.*"

Reply:
changed in vdr-permashift and vdr-pvr350